packages/fxa-auth-server/lib/senders/emails/partials/metadata.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const metaData = `
+  <% if(oneClickLink){ %>
```

Comment:
Nit: we'll want to maintain JS syntax consistency, since prettier doesn't pick up on ERB (at least not yet): `<% if (oneClickLink) { %>`